tests/xtro-sharpie/iOS-ExposureNotification.todo

```diff
@@ -0,0 +1,68 @@
+!missing-enum! ENAuthorizationStatus not bound
```

Comment:
might be easier to ask xtro Makefile to skip this framework (in case of future changes, new betas or Xcode)

Reply:
I wonder if we should keep it as a 🦜 I am not sure if just the wiki is enough, OTOH it might be just a short lived framework